src/App.js

```diff
@@ -81,6 +81,7 @@ export default function App() {
         const data = await res.json();
         setSupportedLanguages(data["supported_languages"]);
         setLoading(false);
+        sessionStorage.setItem('id', Date.now());
```

Comment:
This would reset the session every time the page is refreshed or this function is called, is that what's intended? I think it might be good add a check that it is not already set before setting it. That way a new session id is only generated when the browser decides the session is over.

Reply:
+1. 

Also worth considering if you want the session to restart when browser decides is the session i.e. mostly likely when you close the browser window. 

Or on every new visit to page in which case you would not need to use session storage, but can simply store Date.now() in a variable. I can see both being valid here.